eng/common/performance/performance-setup.ps1

```diff
@@ -1,8 +1,9 @@
 Param(
     [string] $SourceDirectory=$env:BUILD_SOURCESDIRECTORY,
     [string] $CoreRootDirectory,
+    [string] $BaselineCoreRootDirectory,
     [string] $Architecture="x64",
-    [string] $Framework="netcoreapp3.0",
+    [string] $Framework="netcoreapp5.0",
```

Comment:
netcoreapp3.1?

Reply:
These are in eng/common.  Should be left as-is.